src/pages/detail/GroupIntroTab.tsx

```
@@ -9,12 +9,15 @@ import "./styles.css";
 import useSWR from "swr";
 import { getNotionPage } from "../../apis/notion";
 import NotionWrapper from "./NotionWrapper";
+import { useParams } from "react-router-dom";
+import { getGroup } from "@/apis/group";
 
 interface GroupIntroTabProps {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)